playbooks/roles/jira/tasks/main.yml

```diff
@@ -0,0 +1,43 @@
+---
+# Jira role
+
+- name: install Jira dependencies
+  apt:
+    pkg:
+      - mysql-client
+
+- name: Ensure commands can be executed as www-data - set shell
+  user:
+    name: www-data
+    shell: /bin/false
```

Comment:
The name doesn't explain why the shell is set to /bin/false.?

Reply:
Task name updated. Thanks.